ci/tasks/download-deployment-artifact.yml

```diff
@@ -0,0 +1,20 @@
+platform: linux
+
+image_resource:
+  type: registry-image
+  source:
+    repository: alpine
+
+inputs:
+  - name: git-release
+    optional: false
```

Comment:
is this not `false` by default?

Reply:
It didn't say in the docs what the default is and then I though this make its very clear to anyone looking at it so I kept it. If you're right then I'm happy to remove it?
https://concourse-ci.org/tasks.html#input-optional

Reply:
yeah, I actually looked at docs and it isn't clear. I just think it is but happy to leave in since it is clearer